packages/lesswrong/server/vulcan-lib/apollo-ssr/renderPage.tsx

```diff
@@ -138,6 +144,24 @@ function isExcludedFromPageCache(path: string, abTestGroups: CompleteTestGroupAl
   return false
 }
 
+const userAgentExclusions = [
+  'health',
+  'bot',
+  'spider',
+  'crawler',
+  'curl',
+  'node',
+  'python'
```

Comment:
Flagging that this overlaps logically with isHealthCheck in `renderPage.tsx` and `botSiteUserAgentRegexesSetting` in `botRedirect.ts`. It would be nice to combine these, I'll try and do so when I add this for us (by adding these as default regexes to botSiteUserAgentRegexesSetting)